### PR TITLE
Change endpoint url issue#73

### LINKED ISSUE #73: Endpoint URL need to be changed temporarily
**Is this a BUG REPORT or FEATURE REQUEST?**:

> Uncomment only one, leave it on its own line: 
>
> /kind bug
/kind feature

**What happened**: Currently there is issue with /v1beta/ nginx conf to redirect to different ports. THis makes all the calls from Dashboard to fail, though the URL works fine from Curl/Postman

**What you expected to happen**: Temporarily change the URL to use /orchestration/<tenant_id>/*. This should be marked as FIXME

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

**Environment**:
- Orchestration(release/branch) version:
- OS (e.g. from /etc/os-release):
- Kernel (e.g. `uname -a`):
- Install tools:
- Others:

### PR DESCRIPTION
**Pull request by kumarashit**
**What this PR does / why we need it**: Change endpoint URL

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #73 

**Special notes for your reviewer**:

**Release note**:

```release-note
```

### CODE REVIEW

File: orchestration/api/instances.py [hunk 1]
```diff
@@ -38,7 +38,9 @@
 # St2 example of data:/v1beta/orchestration/instances -H
 # "Content-Type:application/json" -d
 # '{"action":"aks.ak_echo_py", "parameters":{"message":"Hello World"}}'
-@instance.route("/v1beta/<string:tenant_id>/orchestration/instances",
+# FIXME: changing the URL to /orchestartion/<tenant_id>/instances,
+# it should be /v1beta/<string:tenant_id>/orchestration/instances
+@instance.route("/orchestration/<string:tenant_id>/instances",
                 methods=['POST'])
```
Comment (wisererik) [inline orchestration/api/instances.py]: /orchestration/<string:tenant_id>/instances?? why change the url?

**Comment by skdwriting** [comment]
@kumarashit , @joseph-v - please hold this merge, till we get confirmation on the solution trial by @wisererik

**Comment by wisererik** [comment]
it's not a issue, will close it.

**Comment by skdwriting** [comment]
handles the issue with a proper handling in PR #117 (dashboard)

**Pull request closed without merging**